tests/generate_test_data.sh

```diff
@@ -3,4 +3,18 @@
 set -e
 
 # Regenerate tests/testdata/expert_models (for various tests).
-experiments/train_experts.sh -r
+# experiments/train_experts.sh -r
```

Comment:
Don't think you want to commit this line.

Reply:
Oops, thanks for catching that!